src/transform.cpp

```diff
@@ -41,8 +47,8 @@ const Tile& Transform::transformTile(Tile& tile, uint16_t extent) {
             for (const auto& r : geom.get<ProjectedRings>()) {
                 TilePoints ring;
                 ring.reserve(r.points.size());
-                for (const auto& p : r.points) {
-                    ring.push_back(transformPoint(p, extent, z2, tx, ty));
+                for (const auto& pt : r.points) {
+                    ring.emplace_back(trans_x(pt.x,extent,z2,tx),trans_x(pt.y,extent,z2,ty));
```

Comment:
Looks like, good catch @mourner !

@springmeyer - can you explain why the above is optimisation ?